modules/doc/content/newsletter/2022_01.md

```diff
@@ -62,6 +62,15 @@ generated by this mesh generator.
       id=quad_peripheral_mesh
       caption=An example peripheral ring (cyan region) generated by `PeripheralRingMeshGenerator`.
 
+### Flux boundary conditions for the Navier Stokes finite volume implementation
```

Comment:
@GiudGiud I don't know how I missed this earlier, but did you mean to put this and your bulleted additions below into the January news?

Reply:
well it technically happened in January

Reply:
If it happened in January, that is fine. Just wanted to double-check.